statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -57,165 +69,155 @@ def ma_rep(coefs, maxn=10):
 
     .. math:: y_t = \mu + \sum_{i=0}^\infty \Phi_i u_{t-i}
 
-    e.g. can recursively compute the \Phi_i matrices with \Phi_0 = I_k
-
-    Returns
-    -------
-    phis : ndarray (maxn + 1 x k x k)
+    The MA coefficients are :math:`\Phi_i` where :math:`\Phi_0 = I_k`.
     """
     p, k, k = coefs.shape
-    phis = np.zeros((maxn+1, k, k))
+    phis = np.zeros((maxn + 1, k, k))
     phis[0] = np.eye(k)
 
     # recursively compute Phi matrices
     for i in range(1, maxn + 1):
-        for j in range(1, i+1):
+        for j in range(1, i + 1):
             if j > p:
                 break
 
-            phis[i] += np.dot(phis[i-j], coefs[j-1])
+            phis[i] += np.dot(phis[i - j], coefs[j - 1])
 
     return phis
 
 
-def is_stable(coefs, verbose=False):
+@deprecate_kwarg('verbose', 'eigenvalues')
+def is_stable(coefs, eigenvalues=False):
     """
-    Determine stability of VAR(p) system by examining the eigenvalues of the
-    VAR(1) representation
+    Test stability of a VAR(p)
 
     Parameters
     ----------
     coefs : ndarray (p x k x k)
+        VAR coefficients
+    eigenvalues : bool
+        Flag indicating to also return eigenvalues
 
     Returns
     -------
     is_stable : bool
-    """
-    A_var1 = util.comp_matrix(coefs)
-    eigs = np.linalg.eigvals(A_var1)
+        Indicator that the maximum eigenvalue is less than 1
+    eigenvals : array, optional
+        Array of eigenvalues of the VAR(1) coefficient in the companion form
 
-    if verbose:
-        print('Eigenvalues of VAR(1) rep')
-        for val in np.abs(eigs):
-            print(val)
-
-    return (np.abs(eigs) <= 1).all()
+    Notes
+    -----
+    Computes the eigenvalues of the coefficient matrix in the the companion
+    form of the VAR(p).
+    """
+    a_var1 = util.comp_matrix(coefs)
+    eigs = np.linalg.eigvals(a_var1)
+    stable = bool((np.abs(eigs) <= 1).all())
+    if eigenvalues:
+        return stable, eigs
```

Comment:
-1 for multiple return

Reply:
Like many of the pep8 improvements, this is not a big deal in itself, but as a pattern around the codebase adds up.